salt/states/archive.py

```diff
@@ -533,7 +553,7 @@ def extracted(name,
         then re-create that directory before extracting. Note that ``clean``
         and ``clean_parent`` are mutually exclusive.
 
-        .. versionadded:: Sodium
+        .. versionadded:: Neon
```

Comment:
not related to this PR, just a mistake from my previous one. I hope it's ok to fix it here too